pkg/cmd/pulumi/import.go

```diff
@@ -904,14 +904,14 @@ func newImportCmd() *cobra.Command {
 				}
 			}
 
-			m, err := getUpdateMetadata(message, root, execKind, execAgent, false, cmd.Flags())
```

Comment:
I have to reorder this so that we grab config before getting update metadata because we want to store the esc env information (retrieved from `getStackConfiguration`) into the metadata